cortex-installer.sh

```diff
@@ -1925,6 +1928,25 @@ function ask_sudo() {
   fi
 }
 
+function prompt_for_usage_stats() {
+  if [ "$CORTEX_ENABLE_TELEMETRY" != "true" ] && [ "$CORTEX_ENABLE_TELEMETRY" != "false" ]; then
+    while true
+    do
+      echo
+      read -p "Would you like to help improve Cortex by anonymously sending error reports and usage stats to the dev team [y/n] " -n 1 -r
```

Comment:
I just checked ubuntu, and they use "[Y/n]" and accept upper or lower case, so let's just do that throughout (I think that is already the behavior in the rest of `cortex-installer.sh`